tests/scripts/bootstrap.sh

```diff
@@ -118,7 +118,7 @@ case "$(uname)" in
         fi
 
         if [ "$BUILD_TARGET" == pretty-check ]; then
-            sudo apt-get install -y clang-format-16 shellcheck
+            sudo apt-get install -y clang-format-19 shellcheck
```

Comment:
Can you please update `install_packages_brew()` to also check and install `19` instead of `16`?

Reply:
Yeah I'll update brew when updating that in openthread :)